zerver/lib/upload.py

```diff
@@ -130,6 +130,16 @@ def get_realm_for_filename(path):
         return None
     return get_user_profile_by_id(key.metadata["user_profile_id"]).realm.id
 
+def delete_message_image_s3(path_id):
```

Comment:
@timabbott I couldnt get it to work the first time so I just tested out the function in a separate script and that worked. Ive uploaded the script here

https://gist.github.com/rahuldeve/8144ab8b651865ca1c35f76cc4d38e93